src/script.ts

```diff
@@ -63,7 +66,33 @@ export interface ScriptConfig extends ScriptReference {
    * - "if-file-deleted": If an input file has been deleted, and before restoring from
    *   cache.
    */
-  clean: boolean | 'if-file-deleted';
+  clean: AstNode<boolean | 'if-file-deleted'> | undefined;
```

Comment:
I like having default values made explicit by the `Analyzer`, so that there's never a mixup over what the default is. `clean` is accessed in two places in `Executor`, so both need the `?? true` now.

Can we preserve that and not add `undefined` here?

Reply:
Yeah, that's reasonable. We don't need a `clean` AST node anyways. Done